src/fbw/src/AdditionalData.h

```diff
@@ -33,4 +33,8 @@ struct AdditionalData {
   unsigned long long realisticTillerEnabled;
   double tillerHandlePosition;
   double noseWheelPosition;
+  unsigned long long syncFoEfisEnabled;
```

Comment:
I think these should be double to be consistent with the previous once above. The only other ULL have a different source. Also the next 3 lines.